{
  "WorkItem": {
    "AffectedComponent": {
      "Name": "",
      "DisplayName": ""
    },
    "ClosedComment": "This should be fixed in our new release, 0.8.2.0. Please give it a try and reactive if you're still seeing this problem. Thanks again for reporting it.",
    "ClosedDate": "2013-01-21T18:41:22.497-08:00",
    "CommentCount": 1,
    "Custom": null,
    "Description": "In TypeScript 0.8.1 (It worked in 0.8.0) getter has to return value, but methods can only throw errors,\n\npublic get name(): string { throw \"Error\"; } // this will cause compilation error\n\npublic test(): string { // This works\n   throw \"Error\"; \n}\n\n(I was using throwing errors to mark abstract getters)",
    "LastUpdatedDate": "2014-01-10T10:34:33.287-08:00",
    "PlannedForRelease": "",
    "ReleaseVisibleToPublic": false,
    "Priority": {
      "Name": "Low",
      "Severity": 50,
      "Id": 1
    },
    "ProjectName": "typescript",
    "ReportedDate": "2012-11-27T11:49:04.893-08:00",
    "Status": {
      "Name": "Closed",
      "Id": 4
    },
    "ReasonClosed": {
      "Name": "Unassigned"
    },
    "Summary": "Getter cannot throw error instead of returning value",
    "Type": {
      "Name": "Issue",
      "Id": 3
    },
    "VoteCount": 1,
    "Id": 499
  },
  "FileAttachments": [],
  "Comments": [
    {
      "Message": "Thanks for reporting this. Marking 'active'.",
      "PostedDate": "2012-11-27T14:21:16.94-08:00",
      "Id": 83131
    }
  ]
}